api/src/api/opportunities_v1/opportunity_routes.py

```diff
@@ -180,7 +180,7 @@
     examples=examples,
 )
 @opportunity_blueprint.output(opportunity_schemas.OpportunitySearchResponseV1Schema())
-@opportunity_blueprint.auth_required(api_key_auth)
+@api_key_multi_auth.login_required
```

Comment:
If we're adding a new auth to these endpoints, can you add some tests for each endpoint with the new auth? I'd say we want at least:
* Happy path - endpoint works with the new token
* Invalid token - endpoint raises a 403